doc-content/enterprise-only/installation/jms-broker-configure_proc.adoc

```diff
@@ -0,0 +1,25 @@
+[id='JMS-broker-configure_{context}']
+= Downloading and configuring AMQ Broker
+
+AMQ Broker enables your applications to communicate with any messaging provider. It specifies how components such as message-driven beans, Enterprise JavaBeans, and servlets can send or receive messages. 
+
+.Procedure
+. Navigate to the https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[Software Downloads] page in the Red Hat Customer Portal (login required), and select the product and version from the drop-down options:
+* *Product: AMQ BROKER*
+* *Version: 7.2*
+. Click *Download* next to *Red Hat AMQ Broker 7.2.1* (`amq-broker-7.2.1-bin.zip`).
+. Extract the `amq-broker-7.2.1-bin.zip` file. 
+. Change directory to `amq-broker-7.2.0-bin/amq-broker-7.2.1/bin`.
+. Enter the following command to create the AMQ Broker user, where `<username>` and `<password>` is a user name and password combination of your choice. COMMENT: What is `mybroker`?
++
+[source]
+----
+create --host $MAIN_NODE_IP --user <username>` --password <password> --require-login mybroker
```

Comment:
please remove references to MAIN_NODE_IP. I used this variable name to make it clear in the QE setup doc, that we're referring to the same ip address. This should be replaced by a placeholder like `<ip-address-or-hostname>` and explanation that this is the ip address or hostname of the server, that the AMQ broker is running on.